### ISSUE TITLE
Question1 about loss define.

### ISSUE THREAD

**Issue by xiejiachen** [TaoRuijie/TalkNet_ASD#31]
codes in https://github.com/TaoRuijie/TalkNet_ASD/blob/main/talkNet.py
line 38.
nloss = nlossAV + 0.4 * nlossA + 0.4 * nlossV

Question 1:  how do you specify 0.4 as the parameter.

Question2:  why nloss defined by sum of these three loss.
As I saw from the your paper, I guess nloss should **only equal to nlossAV**, like figure 2.
As you also only use AV transformer to do prediction at https://github.com/TaoRuijie/TalkNet_ASD/blob/main/talkNet.py line 61

**Comment by TaoRuijie** [comment]
Q1: We follow the original AVA paper, which choose 0.4.Check here:https://arxiv.org/pdf/1901.01342.pdf, Page 7. Similar setting can be found in this paper: https://openaccess.thecvf.com/content_CVPR_2020/papers/Alcazar_Active_Speakers_in_Context_CVPR_2020_paper.pdf, page 5, they choose 1.0.

Q2: As we mentioned in Q1, in many existing works already used 3 losses togethor. Adding this into the figure will make it difficult to understand, so we ignore that.

From my experments, this setting can improve the performance by a small margin (about 0.5% mAP)

**Comment by TaoRuijie** [comment]
Also for prediction, in our work we use the outsAV only. Three losses are only used for training as what I remember.
